### ISSUE TITLE
Filebot WebHook

### ISSUE THREAD

**Issue by Housemann** [jlesage/docker-filebot#52]
Hello thanks for the great container. 

I have installed with "AMC_INSTALL_PKGS" the package "curl". I want to send me data via a webhook what filebot has renamed. Is it possible to do this?

**Comment by jlesage** [comment]
You mean how to pass data to your webhook ?  Do you know how it expects the data ?

**Comment by Housemann** [comment]
Exactly. how can i send the old and new filename to the webhook? something like this: 
http://user:pass@10.10.10.111:3777/hook/test?OldFileName=OldMovieFile&NewFileName=NewMovieFile

**Comment by Housemann** [comment]
I am a little further. when i make this: 
--def exec="curl http://user:pass@10.10.10.111:3777/hook/test?FilenameOriginal="\"{original}\" it works great. The Hook sends me My.First.Movie.1999
If i try this:
 --def exec="curl http://user:pss@10.10.10.111:3777/hook/test?FilenameOriginal="\"{original}\""&FilenameNew="\"{fn}\" 
The Filebot log tells me: 
Bad exec value: curl http://user:pass@10.10.10.111:3777/hook/test?FilenameNew="{fn}"

What is wrong?

**Comment by Housemann** [comment]
Here is the solution
--def exec="curl -G -v "http://user:pass@10.10.10.111:3777/hook/test" --data-urlencode "FileNameOriginal=\"{original}\"" --data-urlencode "FileNameNew=\"{fn}\"""
